src/GraphQL/Validation/Rules.Custom/InputFieldsAndArgumentsOfCorrectLength.cs

```diff
@@ -18,7 +18,7 @@ namespace GraphQL.Validation.Rules
     /// Please copy the newer <see cref="PatternMatchingDirective"/> and <see cref="PatternMatchingVisitor"/>
     /// when designing new custom directives that can be used to validate input fields and arguments.
     /// </remarks>
-    public class InputFieldsAndArgumentsOfCorrectLength : IValidationRule, IVariableVisitorProvider
+    public class InputFieldsAndArgumentsOfCorrectLength : ValidationRuleBase
```

Comment:
This is the sample custom validation rule included with GraphQL.NET that uses two tie-in points within the updated `IValidationRule` interface.  Both previously and now, it only requires a single `.AddValidationRule<InputFieldsAndArgumentsOfCorrectLength>()` line of code to install.  The main difference is that previously the `IVariableVisitorProvider` was hidden, in the sense that you'd have to know how GraphQL.NET works to know that it checks if `rule is IVariableVisitorProivder` and make use of its functionality.  Now, anyone that inspects `IValidationRule` can see the method on the interface.